ops/triton/triton_ragged_hstu_attention.py

```diff
@@ -409,6 +584,7 @@ def _ragged_hstu_attn_fwd(  # noqa C901
     BLOCK_N: tl.constexpr,
     max_attn_len,
     HAS_MAX_ATTN_LEN: tl.constexpr,
+    enable_tma: tl.constexpr,
```

Comment:
also add k_desc and v_desc?

Reply:
Hmm, for this one I did not see a k/v_desc on the top level fwd kernel:

https://github.com/manman-ren/generative-recommenders/blob/c4cef7e41a97ff912ccca6b77be46117228b4260/ops/triton/triton_ragged_hstu_attention_tma.py#L381

Seems you were just using K. Should I still add one?

Reply:
Yeah my version only has K, V, but we pass in k/v_desc for them (https://github.com/manman-ren/generative-recommenders/blob/c4cef7e41a97ff912ccca6b77be46117228b4260/ops/triton/triton_ragged_hstu_attention_tma.py#L771). Since your version supports both tma and no-tma, should we pass both K/V and k_desc/v_desc?